tests/test_parameters.py

```diff
@@ -49,7 +50,7 @@ def run_custom_parameters_test(self, agent_policies, expected_result, delete_log
                 custom_agent_index = all_agent_index - number_of_init_agents  # identify which custom agent we are on
                 expected_result_dict = expected_result[custom_agent_index]
                 for key, value in expected_result_dict.items():  # for each custom parameter to check
-                    np.testing.assert_equal(
+                    np.testing.assert_almost_equal(
```

Comment:
What changed such that this test result can no longer be equal?

Reply:
not sure, set it back. these values are simply assigned, so should be no rounding issues.